cmd/neofs-node/config.go

```diff
@@ -441,7 +441,8 @@ type cfgContainer struct {
 }
 
 type cfgNetmap struct {
-	wrapper *nmClient.Client
+	wrapperNoNotary *nmClient.Client
```

Comment:
It sucks. Really, really, really sucks. You can call it `wrapperDirect` at least. But semantically it's just a `ContractReader` part of the deal from its use. And I'm not sure how it can affect anything (these are test invocations).

Reply:
we can make a single client that will be try-notary'ed if you want. i just remember how it was done before in some cases (we may want to combine notary and non-notary calls for the same contract)

Reply:
every client is notary in fact, dropped